frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx

```diff
@@ -529,6 +513,9 @@ export const UploadPVCPage: React.FC<UploadPVCPageProps> = (props) => {
   const handleFileChange = (value, filename) => {
     setFileName(filename);
     setFileValue(value);
+
+    setFileNameExtension(/[.][^.]+$/.exec(filename)?.toString());
+    setIsFileRejected(false);
```

Comment:
when is isFileRejected true? @yaacov

Reply:
it is set by the `UploadPVCForm` when file extension is unexpected, but is not reset back on field clear, this line reset it back when the offending file is cleared but no new file is selected.